libraries/core_libs/network/rpc/Test.cpp

```diff
@@ -240,49 +135,6 @@ Json::Value Test::get_packets_stats() {
   return res;
 }
 
-Json::Value Test::get_node_version() {
```

Comment:
I think we wanted to have this, so we can check user node versions easily

Reply:
I don't think we need this. Because we have [this](https://github.com/Taraxa-project/taraxa-node/pull/1756/files#diff-475024e647753c48d72e3a4a796cf29cd7ec65a528f542d0aeed9b3e758daed7R44) and [this](https://github.com/Taraxa-project/taraxa-node/pull/1756/files#diff-475024e647753c48d72e3a4a796cf29cd7ec65a528f542d0aeed9b3e758daed7R70). Please check `RPC.md`